ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OGridSearch.scala

```diff
@@ -414,6 +417,8 @@ trait H2OGridSearchParams extends DeprecatableParams {
   private val selectBestModelDecreasing = new BooleanParam(this, "selectBestModelDecreasing",
     "True if sort in decreasing order accordingto selected metrics")
   private val foldCol = new NullableStringParam(this, "foldCol", "Fold column name")
+  private val convertUnknownCategoricalLevelsToNa = new BooleanParam(this, "convertUnknownCategoricalLevelsToNa", "Convert unknown" +
+    " categorical levels to NA during predictions")
```

Comment:
nit: What about a different comment?
```If set to 'true', the model converts unknown categorical levels to NA during making predictions.```

Is this also related to the training phase?